.github/workflows/test.yml

```diff
@@ -627,7 +627,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [macos-latest, windows-latest]
-        node: [16]
+        node: [14]
```

Comment:
@millsp  Technically, we only see windows failing on Node v16? not macOS?

Random thing
`NODE_OPTIONS: '--max-old-space-size=8096'` is currently set (for client only)
and we actually have less RAM available 😄 https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

> Hardware specification for Windows and Linux virtual machines:
>     2-core CPU
>     7 GB of RAM memory
>     14 GB of SSD disk space
> 
> Hardware specification for macOS virtual machines:
>     3-core CPU
>     14 GB of RAM memory
>     14 GB of SSD disk space

Reply:
Thank you. Let me try that again!

Reply:
Trying with `6512` as recommended in the [Node.js manual](https://nodejs.org/api/cli.html#--max-old-space-sizesize-in-megabytes).

Reply:
It fails the same :/